DependencyInjection/eZDemoCompilerPass.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\DemoBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+class eZDemoCompilerPass implements CompilerPassInterface
```

Comment:
I'd name it after what it does, not after what it belongs to.

`PrivacyCookiePass` ?